jib-core/src/main/java/com/google/cloud/tools/jib/image/json/JsonToImageTranslator.java

```diff
@@ -68,7 +68,7 @@
    */
   public static Image<Layer> toImage(V21ManifestTemplate manifestTemplate)
       throws LayerPropertyNotFoundException {
-    Image.Builder<Layer> imageBuilder = Image.builder();
+    Image.Builder<Layer> imageBuilder = Image.builder(V22ManifestTemplate.class);
```

Comment:
Just checking: `manifestTemplate` is `V21ManifestTemplate`, but here you are using `V22ManifestTemplate`. Correct?

Reply:
Yeah, the parameter passed into the image builder is the target image format, and we don't build v21 images.